### PR TITLE
feat: open im modal on click

### PR DESCRIPTION
**Pull request by luqven**
This PR makes it so that when a product image is clicked or the `add image` button is clicked, a modal with the imgix IM opens.

https://user-images.githubusercontent.com/16711614/150235136-ef79b8da-c4a4-4f26-b937-48a04cafb0a1.mov

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #184** 
    * **PR #185** 
      * **PR #187** 
        * **PR #186**  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/ExtensionApp.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React from "react";
+import { App as BreakoutApp } from "../components/BreakoutApp";
 import "../styles/App.css";
+import "../styles/index.css";
```
Comment (luqven) [inline frontend/src/components/ExtensionApp.tsx:4]: These are the styles that `BreakoutApp` relies on for fonts, etc.

Comment (frederickfogerty) [reply frontend/src/components/ExtensionApp.tsx:4]: Definitely think we should refactor these into something modular as much as possible, and reduce global styles as much as possible. 

Also, it seems that index.css just imports from App.css, so is index.css actually necessary?

**Review by frederickfogerty — COMMENTED** [review]
LGTM so far.

File: frontend/src/components/ExtensionApp.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 import { IImgixCustomAttributeValue } from "../types/imgixSF";
+import Modal from "./layouts/Modal";
 import { ProductPageImages } from "./ProductPageImages";
 
 export type ISidebarAppProps = {
```
[the remark this answers is not in the document]
Comment (luqven) [reply frontend/src/components/ExtensionApp.tsx:9]: 🤦🏼‍♂️ 100%

File: frontend/src/components/ExtensionApp.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/ExtensionApp.tsx:4]: Gonna double check the `index.css` import. Distinctly remember needing for the fonts but could be I wasn't importing `App.css` at the time.

**Comment by luqven** [comment]
Closing this as UI and data flow have changed

**Pull request closed without merging**